spec/features/users/account/logout_spec.rb

```diff
@@ -0,0 +1,36 @@
+RSpec.describe "Déconnexion" do
+  let(:user) { create(:user) }
+  let(:motif) { create(:motif, bookable_by: :everyone, organisation: organisation) }
+  let(:organisation) { create(:organisation) }
+  let(:lieu) { create(:lieu, organisation: organisation) }
+
+  let!(:plage_ouverture) { create(:plage_ouverture, :weekdays, first_day: 1.month.from_now, motifs: [motif], lieu: lieu, organisation: organisation) }
+
+  it "fonctionne après une prise de rdv" do
+    visit public_link_to_org_path(organisation_id: organisation.id, org_slug: organisation.slug)
+    click_on motif.name
+    click_on lieu.name
+    first(:link, "11:00").click
+
+    fill_in("Adresse email", with: user.email)
+    fill_in("Mot de passe", with: user.password)
+    find("input[value='Se connecter']").click
+
+    click_button("Continuer")
+    click_button("Continuer")
+    click_link("Confirmer mon RDV")
+    expect(page).to have_content("Votre RDV")
+
+    click_on "Déconnexion"
+
+    visit public_link_to_org_path(organisation_id: organisation.id, org_slug: organisation.slug)
+    click_on motif.name
+    click_on lieu.name
+    first(:link, "11:00").click
+
+    # On vérifie que les infos de l'utilisateur n'ont pas été gardées dans la session
+    expect(page).to have_content("Vous devez vous connecter ou vous inscrire pour continuer.")
+
+    expect(page).not_to have_content(user.first_name)
```

Comment:
```suggestion
    expect(page.body.include?(user.first_name)).to be false
```

cette assertion ne fonctionnait pas chez moi pour verifier que c’était cassé avant, j’imagine que c’est parce que c’était dans une value d’input

Reply:
oups, c'est vrai. J'ai vu ça un peu tard, mais je l'intègre à https://github.com/betagouv/rdv-service-public/pull/5352 (puisque je refacto le code qui fait ce fix dans cette pr)